presentation/build.gradle.kts

```diff
@@ -64,6 +64,7 @@ dependencies {
     implementation(libs.bundles.compose)
     implementation(libs.lifecycle.runtimeKtx)
     implementation(libs.android.hilt)
+    implementation("androidx.core:core-splashscreen:1.0.0-rc01")
```

Comment:
hey @keithchad Can you define this dependency like the others

Basically:
1. Look for `libs.versions.toml`, and define this dependency there (top of the file [versions] defines the version, then add it under the [libraries] section) - see how other dependencies are setup
2. Then use it on the presentation build file like ` implementation(libs.androix.splashscreen)`

Reply:
Okay. Working on it.

Reply:
I have updated both Gradle files with the right implementation.

Reply:
Thank you